libazureinit/src/lib.rs

```diff
@@ -3,6 +3,7 @@
 
 pub mod error;
 pub mod goalstate;
+pub mod http;
```

Comment:
Sorry, should have caught this last time, but I noticed the HTTP constants aren't used in any public interface, so it would be best to not expose them publicly in the API. You can make them visible to everything within `libazureinit` (and not available outside it) by changing this to 
```suggestion
pub(crate) mod http;
```

That way we minimize the surface of our public API.